marvel-app/features/alamofire/AlamofireViewModel.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  AlamofireViewModel.swift
+//  marvel-app
+//
+//  Created by Michal on 05/08/2022.
+//
+
+import Foundation
+import Combine
+
+class AlamofireViewModel: ObservableObject{
+
+    @Published var comics: ComicDataWrapper?
+
+    var alamofireProtocol: AlamofireProtocol
+    private var cancellables: Set<AnyCancellable> = []
```

Comment:
Please init cancellables as `Set<AnyCancellable>()`. Also consider naming this variable as bag because its the default name for it.